meso_tools/NAS_tools.py

```diff
@@ -0,0 +1,96 @@
+#Tool currently returns list of session filepaths in storage that are candidates for deletion
+#Currently only published/passed sessions.  Should update to include failed and incomplete sessions 
+#that are associated with "published" mouse_ids
+
+import typing
+import requests
+def credential_read(cred_path: str):
+    """Takes txt doc containing NAS credentials and outputs a dictionary containing the relevent info.  
+    Expected format for txt doc:
+    
+    # Device access credentials
+    SYNOLOGY_IP=xx.xxx.xx.xx:xxxx
+    SYNOLOGY_PORT=xxxx
+    SYNOLOGY_USERNAME=x
+    SYNOLOGY_PASSWORD=x
+
+    Args:
+        cred_path (str): relative string path of txt document containing NAS credentials
+    """    
+
+    with open(cred_path) as f:
+        credentials = f.readlines()
+    lines =[]
+    for line in credentials[1:]:
+        line_split = line.split('=')
+        for items in line_split:
+            item = items.split('\n')[0]
+            lines.append(item)
+    keys = lines[::2]
+    values = lines[1::2]
+    res = {keys[i]: values[i] for i in range(len(keys))}
+    return(res)
+
+def nas_query(credentials: dict, nas_name: str, release_list: str):
```

Comment:
add output type using 
def function_name(input1 : str, input2 : int) -> int:
   body of the function
   return output (integer)

Reply:
done